src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/BlockPlacer.java

```diff
@@ -182,6 +186,10 @@ private void placeSlimefunBlock(SlimefunItem sfItem, ItemStack item, Block block
     private void placeBlock(ItemStack item, Block facedBlock, Dispenser dispenser) {
         BlockPlacerPlaceEvent e = new BlockPlacerPlaceEvent(dispenser.getBlock(), item, facedBlock);
         Bukkit.getPluginManager().callEvent(e);
+        
+        if(!dispenser.getInventory().getViewers().isEmpty()){
```

Comment:
Formatting, please adhere to our code style guidelines, there should be a space after the `if` and before the curly bracket `{`